src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -683,23 +687,29 @@ function BaseSelectionList<TItem extends ListItem>(
      * @param timeout - The timeout in milliseconds before removing the highlight.
      */
     const scrollAndHighlightItem = useCallback(
-        (items: string[], timeout: number) => {
+        (items: string[]) => {
             const newItemsToHighlight = new Set<string>();
             items.forEach((item) => {
                 newItemsToHighlight.add(item);
             });
             const index = flattenedSections.allOptions.findIndex((option) => newItemsToHighlight.has(option.keyForList ?? ''));
-            updateAndScrollToFocusedIndex(index);
+            scrollToIndex(index);
```

Comment:
This change will remove `setFocusedIndex`. Could you please explain the reasoning behind this change?

Reply:
Previously we highlight the item using the focused index and `itemsToHighlight`. Focused style will have a different background then what we want and it's not animated. Also, instead of having 2 different way of highlighting it, we can just use the `itemsToHighlight`. The focused index will also be reset to -1 after the delay.
https://github.com/Expensify/App/blob/445fc4258bc4c7e0d43e833c724587b917222cda/src/components/SelectionList/BaseSelectionList.tsx#L692-L700

Reply:
Got it, Thanks for your feedback!